ts/src/program/index.ts

```diff
@@ -79,6 +79,7 @@ export class Program<IDL extends Idl = Idl> {
    * });
    * ```
    * @deprecated
+   * Use program.methods.<method>(...args) instead of the rpc call
```

Comment:
Hmmm? Shouldn't that be?
```suggestion
   * Use program.methods.rpc.<method>(...args) instead of the rpc call
```